### ISSUE TITLE
Hülle

### ISSUE THREAD

**Issue by phiweger** [lucaskassel/flutter_mate_shader#2]
@lucaskassel adding this as issue/ task, so we don't forget. What remains to do here, exactly?

**Comment by lucaskassel** [comment]
Es gibt eigentlich nur noch zwei Dinge zu tun:
1. Eine schimmernde Hülle hinzufügen, um visuell etwas mehr Tiefe zu erzeugen
2. Die Farben final mit dem Design der App abstimmen

Optional könnten wir noch weitere Interaktionen bzw. Zustände hinzufügen. Aktuell gibt es nur Pulse (also ein kurzes Pulsieren von Mate). Denkbar wären außerdem noch Start/ Idle (gibt es im Prinzip schon, aber nicht separat) und Stopp (es wurde mal gesagt, dass Mate u.a. in der Taskbar ruhen soll) und ein Zustand der das Verarbeiten einer Anfrage anzeigt.

**Comment by phiweger** [comment]
implementierst du das noch @lucaskassel ?

**Comment by lucaskassel** [comment]
@phiweger Ja, kommt bis morgen früh mit Ausnahme des optionalen Teils. Da müssten wir uns vorher noch mal abstimmen, da das abhängig von der Implementierung von Mate in der App ist.
